packages/back-end/src/types/Integration.ts

```diff
@@ -331,6 +346,25 @@ export interface InformationSchemaTablesInterface {
   informationSchemaId: string;
 }
 
+export interface DimensionMetadataResult {
+  dimension: string;
+  dimensionValues: { name: string; percent: number }[];
+}
+
+export interface DimensionMetadataInterface {
```

Comment:
same suggestion about naming (metadata too broad?)

Reply:
Ok, renaming to dimensionSlices where important. I still keep it as `dimensionMetadata` in the `ExposureQueryInterface` because we may add additional metadata to that object.